### DISCUSSION TITLE
Обсуждение на Великом и Могучем

### DISCUSSION THREAD

**Discussion by Bluesboy** [rms-support-letter/rms-support-letter.github.io#1409] [General]
Здесь ведем обсуждение на русском языке

**Comment by AnimusPEXUS** [comment]
нужно ещё создать петиции, чтобы из проектов, которые возникли на RMSа, изгнали экстремистов.

**Reply by TimurC1** [reply]
Тоже об этом думал. Лидера GNOME точно сразу выгнать, организация же подконтрольна GNU

**Comment by TimurC1** [comment]
Перемещаюсь сюда тоже, из issue :)

**Comment by Potapov-Ilya** [comment]
Я нифига не понял как подписаться за RMS. Подскажите?

**Reply by alesharik** [reply]
В `_data/signed` создаёте файл _ваш_ник_.yaml, туда записываете:
```yaml
name: <ваша фамилия и имя на английском, без скобок>
link: <ссылка на ваш личный сайт или на профиль github, без скобок>
```

**Reply by ghost** [reply]
В README ведь написано всё. Создаёшь форк
```
create a file in _data/signed/ folder named <username>.yaml with the following content:

name: <your name here>
link: <link to your profile or site>

Without the <>.

Example:

name: Example name
link: https://example.com/

Don't use <> in this file, as well as non-ascii symbols in file name. If you are able to, please use your real name and add projects and affiliatied organizations in parentheses.

When youre done, create a pull request.
```

**Reply by Potapov-Ilya** [reply]
Всем спасибо, получилось.

**Comment by pda0** [comment]
Поставить подпись - не единственное, что можно сделать.
https://www.opennet.ru/openforum/vsluhforumID3/123704.html

**Comment by ghost** [comment]
https://lists.debian.org/debian-vote/2021/03/msg00083.html

вот что творится в Дебиане

**Reply by AnimusPEXUS** [reply]
?е?ропа и её лайф меттерсы - знатны в любви к христовым походам.
последний самый известный был в 41-45

**Comment by Bluesboy** [comment]
Может кто-нибудь переведет само письмо на русский?

**Reply by alesharik** [reply]
Так уже в README_RU.md

**Reply by Bluesboy** [reply]
это ридми, нужно тело письма.

**Reply by Bluesboy** [reply]
#2076 Вот сделал свой перевод, но кто-то успел раньше меня. Посмотрим, чей лучше.

**Comment by JantoreRew** [comment]
Как его отношение к ЛГБТ и противоположному полу вообще относится к лицензии GPL?

**Reply by Bluesboy** [reply]
Никакого, в том то и суть.

**Reply by ghost** [reply]
RMS, вроде, не является ярым женоненавистником, оклеветали его

**Comment by Bluesboy** [comment]
Так, сделали перевод, теперь как его добавить, чтобы он был виден всем?

**Reply by Bluesboy** [reply]
Готово, ребята!

**Comment by gorshunovr** [comment]
Новостным сайтам напишите на редакционные почтовые ящики с предложением опубликовать новость или заметку. Пока только на CNews.ru вышла (однобокая) новость. Хабр, опеннет, ЛОР не в счёт, так как это айтишные ресурсы.

Из английской википедии удаляют ссылки на это письмо из-за отсутствия независимых источников.

**Comment by ghost** [comment]
Ответ RMS на письма в FSF:

Thank you for supporting me with the FSF.
You can give me more support by joining the FSF as an
associate member.  I hope you will do that.

Another thing that will help me is to post on public forums in support.

You can also encourage other people to do all these things.
But they should write only to info@fsf.org, not to directors@fsf.org.

The directors have got piles of mail already, and there is no need to add
to the piles.

Thank you.

-- 
Dr Richard Stallman
Chief GNUisance of the GNU Project (https://gnu.org)
Founder, Free Software Foundation (https://fsf.org)
Internet Hall-of-Famer (https://internethalloffame.org)

**Reply by Bluesboy** [reply]
Слезы наворачиваются

**Reply by gorshunovr** [reply]
Задонатил и подписался!

**Comment by Bluesboy** [comment]
https://odysee.com/@Lunduke:e/DefenseofStallman:9

**Reply by pda0** [reply]
Он так забавно произносит "free software foundation". Я сылшу "free suffer foundation".

**Reply by TimurC1** [reply]
> Он так забавно произносит "free software foundation". Я сылшу "free suffer foundation".

Javascript Code и Non-Free Software еще смешнее. Ладно, шучу :)

**Comment by Bluesboy** [comment]
Украл с опеннета:

>вот до того, как выгнали Столлмана: https://www.fsf.org/patrons/fy2018
а вот после: https://www.fsf.org/patrons/fy2020

**Reply by aleksejrs** [reply]
2019 пропустили: https://www.fsf.org/patrons/fy2019

**Reply by podvornyakva** [reply]
2020. Нет красной шапки. А шо такое? Столлман же ушёл. А в чём дело? Короновирус головного мозга?
Sleaky bastards.

**Comment by ghost** [comment]
https://www.debian.org/vote/2021/vote_002

**Reply by leveldev** [reply]
может написать на dtf.ru? Разместить статью или заметку

**Reply by TimurC1** [reply]
Не, на dtf не надо, а на вот еще два других ресурса питерского "Комитета" 
TJournal и vc.ru было бы неплохо. Хотя словим один хейт
> может написать на dtf.ru? Разместить статью или заметку

**Comment by Bluesboy** [comment]
Откопали в архиве (-:
http://web.archive.org/web/20210326054602/https://github.com/rms-open-letter/rms-open-letter.github.io/issues/2250

**Reply by JohnKazitsin** [reply]
Я так понимаю ныне это сообщение удалено чтобы не позориться?

**Reply by systemdfan** [reply]
> Откопали в архиве (-:
> http://web.archive.org/web/20210326054602/https://github.com/rms-open-letter/rms-open-letter.github.io/issues/2250

А что там было?  Я лично ничего не вижу. Они это каким-то образом их вебархива потёрли?

**Reply by Bluesboy** [reply]
Проверил, все открывается....
Но вот на всякий случай:
![image](https://user-images.githubusercontent.com/8276862/112763003-158d5500-901c-11eb-99bf-745446a50a1e.png)

**Reply by systemdfan** [reply]
@Bluesboy, ха, а на этот раз у меня открылось. Но всё равно спасибо.

**Reply by ghost** [reply]
Фреймы разрешены?

**Comment by TimurC1** [comment]
Кстати, в репо против Столлмана на последних строчках есть просто Z. Буква. Вот так подделывают голоса, у нас все честно с ссылками на профиль GH

**Reply by pda0** [reply]
Строго говоря это не обязательно. У нас тоже ники принимают. Можно даже поискать в коммитах чей это был pull request.

**Reply by gorshunovr** [reply]
Всех этот загадочный _Z_ смущает. На самом деле это @purringChaos, вот [этот PR](https://github.com/rms-open-letter/rms-open-letter.github.io/pull/1944).

**Reply by TimurC1** [reply]
> Всех этот загадочный _Z_ смущает. На самом деле это @purringChaos, вот [этот PR](https://github.com/rms-open-letter/rms-open-letter.github.io/pull/1944).

Вы случайно вызвали @purringChaos и его заклинило :) вот видите, не люди подписывают письма против Ричарда, а машины корпоратов. Боты:)

**Reply by TimurC1** [reply]
> Строго говоря это не обязательно. У нас тоже ники принимают. Можно даже поискать в коммитах чей это был pull request.

У нас ссылки все указывают. У них ни одной ссылки, только юзернеймы у нескольких, и то непонятно, на GH или где.

**Comment by TimurC1** [comment]
Небольшой оффтопик. Каким foss пользуетесь? Какие планы на будущее для foss в России по вашему мнению?

**Reply by Bluesboy** [reply]
Linux, естественно, ну и окружение, легче сказать, что у меня закрытого стоит сейчас. Пожалуй, только Slack и Телега.

**Reply by TimurC1** [reply]
> Linux, естественно, ну и окружение, легче сказать, что у меня закрытого стоит сейчас. Пожалуй, только Slack и Телега.

Telegram – клиент открытый, но сервер закрытый. Но сервер не проверить все равно, хотя можно было бы захостить свой при открытом сервере

**Reply by systemdfan** [reply]
> Telegram – клиент открытый, но сервер закрытый. Но сервер не проверить все равно, хотя можно было бы захостить свой при открытом сервере.

А Дуров утверждает что публикация кода серверной части не улучшит безопасность, потому что, в отличие от кода клиентов, нет возможности проверить, выполняется ли тот же код на серверах. [Пруф](https://t.me/durovschat/515221)

**Reply by Bluesboy** [reply]
Мало ли что он утверждает. А что клиент, якобы, с открытыми исходниками, так это ни о чем не говорит. Телега требует твой телефон, чтобы ей пользоваться, это уже достаточно.

**Reply by systemdfan** [reply]
Да я не говорю что он прав. Просто забавно как он умело манипулирует юзерами. И главное же ему кто-то верит.

**Comment by chaoticryptidz** [comment]
hello yes you may have some concern but we both have mononyms, rather than traditional names, these are both of our legal names.On 27 Mar 2021 7:35 pm, Roman Gorshunov ***@***.***> wrote:

Всех этот загадочный Z смущает. На самом деле это @purringChaos, вот этот PR.

**Reply by gorshunovr** [reply]
I understand it, no worries. Just have been explaining to everyone interested that Z is not a mistype and not a fake name.
Peace!

**Comment by chaoticryptidz** [comment]
we are very much against what RMS is doing thus we signed the open letter against RMS and the FSF's actions on keeping RMS on boardOn 27 Mar 2021 7:54 pm, Kit ***@***.***> wrote:hello yes you may have some concern but we both have mononyms, rather than traditional names, these are both of our legal names.On 27 Mar 2021 7:35 pm, Roman Gorshunov ***@***.***> wrote:

Всех этот загадочный Z смущает. На самом деле это @purringChaos, вот этот PR.

**Reply by TimurC1** [reply]
Go outside this topic!

**Comment by systemdfan** [comment]
Я обсуждал письмо в поддержку Столлмана в телеграм чате Антона павленко, за что получил бан в чате и бан в телеграме за "спам". Сегодня в час дня бан должны снять. Вот такие дале. Не пользуйтесь проприетарной заразой, друзья. 
![Снимок экрана в 2021-03-28 10-55-39](https://user-images.githubusercontent.com/81460388/112745924-71bc8e80-8fb4-11eb-92ba-bc5dcfedfb35.png)
![Снимок экрана в 2021-03-27 12-58-13](https://user-images.githubusercontent.com/81460388/112745926-72edbb80-8fb4-11eb-8739-fd92772c78f5.png)

**Reply by AnimusPEXUS** [reply]
меня, в своё время, даже в гугле банили за своё мнение.

по сути то, если подумать, крымчан тоже забанили за своё мнение. (я не призываю развивать тему крыма, просто по факту)

**Reply by systemdfan** [reply]
Крымчан как раз github банил, кстати.

**Reply by Oxyd76** [reply]
Ну всякие депрессии с выгоранием у человека. (Я про Антона.)

**Reply by veselcraft** [reply]
как будто модераторы там заядлые ненавистники рмс...

**Comment by chaoticryptidz** [comment]
lol im no bot haha, what the fuck are y'all smoking On 27 Mar 2021 9:14 pm, Timur ***@***.***> wrote:



Всех этот загадочный Z смущает. На самом деле это @purringChaos, вот этот PR.



Вы случайно вызвали @purringChaos и его заклинило :) вот видите, не люди подписывают подписи против Ричарда, а машины корпоратов. Боты:)

**Comment by AnimusPEXUS** [comment]
так. мне надоело.
давайте формировать новую бригаду по поддержке и разработке FSF и GNU.
я готов включиться в любой момент по C/C++ - но лучше D (можно и Go/Rust - но имхо этим радужным поделиям тут не место)
есть только небольшая загвоздка: нужно финансирование, хотя бы 60 тыр в месяц - минимум 30 тыр, чтобы с голоду не сдохнуть. нельзя позволить проектам загнуться!

зы: не нравится Сталлман и его видение - нихай валят. пора слить протухшую кровь.
ззы: а если эти сквоттеры таки выпхают (что незаконно) Сталлмана из его собственных проектов, тогда сами форкнемся и сделаем всё по-новой, как когдата Бацка Сталлман сделал.

**Comment by gorshunovr** [comment]
Абу за Столлмана:
https://github.com/rms-support-letter/rms-support-letter.github.io/issues/837#issuecomment-809339758
https://t.co/4OYqwQaSbY
https://t.co/riOcQVEcoQ

**Reply by gorshunovr** [reply]
Спасибо, @abuabu!

**Reply by systemdfan** [reply]
> Спасибо, @abuabu!

Это реально настоящий Абу? Тот самый капитан Ever Given, консультант из остина и боец и**л? Спасибо товарищу Xi!  Партия желает  @abuabu большой нефритовый стержень и одна кошка-жена! 

>Абу благословил этот пост

**Comment by trufanov-nok** [comment]
Оффтоп, наверное. Понятно, что Столлман много чего наговорил за последние N-лет. И о нем много чего нарассказывали. Читал недавно CNews, мне кажется они скопипастили тескты противников Столлмана не вдаваясь в подробности... Потому что их больше. Так почему бы не воззвать к одной из вечных демократических ценностей - дебатам! У англосаксов есть целая школа дебатов. Пусть Столлман выйдет раз на раз с уполномоченным противной стороной человеком, и вступит в дебаты по фактам. А мы послушаем и рассудим.  Может предложить Столлману вызвать противников на дебаты перед лицом коммьюнити? А то мы впёрлись в эту голосовалку - а толку с нее, новой информации немного. Пусть вызывает противников на дебаты, и сам давит их.

**Reply by Bluesboy** [reply]
Они на это не пойдут — это кучка трусов, чья позиция ничем не подкреплена и они **сами** об этом знают

**Reply by trufanov-nok** [reply]
Если это так, то они сами обнажат свою слабость. С моей точки зрения это win-win ситуация.

**Reply by Bluesboy** [reply]
Они перешли все границы, очевидно, что что бы FSF не сделал, какое бы решение не принял, они будут недовольны. Таким образом, они загнали FSF в угол и отступать нам некуда. Значит придется идти на конфронтацию, а её петушки не выдержат, на то они и петушки.

**Reply by hinell** [reply]
Дебаты это как если бы ты содержал FSF и мог бы понаблюдать за шоу. Тут совсем иная ситуация. Кучка каких-то МИТовских сумашедших решила макнуть в говно вдхоновителя и основателя движения СПО опираясь на какие-то смутные обвинения в вещах никак не связанных с его деятельность в FSF.

Это проста наглая и откровенная попытка завалить FSF. Мотивов особо никто не скрывает. Всё остальное уже плохо сформуированные предлог.

**Reply by systemdfan** [reply]
Ага, знаем какой у них [уровень дебатов](https://youtu.be/zH_ZryBfCtU?t=23l) - на любые аргументы будут перебивать и повторять неустанно свои мантры.

**Comment by podvornyakva** [comment]
Дратуте.

**Reply by Bluesboy** [reply]
И вам не хворать!

**Comment by AnimusPEXUS** [comment]
интересно: https://github.com/rms-support-letter/rms-support-letter.github.io/discussions/5002 - федоровская альфа-служанка с ЛОРа - не хочет ли поведать нам детали истории, или она только на Сталлмана наезжать может?

**Comment by ADarsiKorushe** [comment]
Ребята, что случилось, что  **rms-open-letter.github.io**  закрыли приём подписей?

> As of April 1, 2021 00:00 UTC we are no longer accepting signatures.

Ссылка на текст https://github.com/rms-open-letter/rms-open-letter.github.io/blob/main/README.md

Или это у них так 1 апреля отметили)

**Reply by pda0** [reply]
Видать решили, что с общественным мнением проигрывают и надо давить из подтишка через компании и организации. Подконтрольная RedHat Fedora включилась в травлю.

**Reply by Bluesboy** [reply]
Это чтобы потом сказать, мол, мы закрыли голосовалку, а эти продолжили, поэтому их больше.

**Reply by 6r1d** [reply]
Бесполезно. В сети есть статистика с учётом времени. Любой, кто понимает в трендах, осознаёт, что здесь произошло.

**Reply by Bluesboy** [reply]
В сети достаточно и доказательств того, что слова РМС вырваны из контекста и извращены. Но разве это кого-то волнует?

**Reply by podvornyakva** [reply]
> Это чтобы потом сказать, мол, мы закрыли голосовалку, а эти продолжили, поэтому их больше.

Сказать можно, доказать нельзя. А без доказательств, все вытекающие из этого заявления, последствия подпадают под УК. Надеюсь мы говорим о правовом государстве и законопослушных гражданах, а не об Украине. =)

Если они готовы заткнуть рот своим сторонникам, закрыв голосовалку - это их право, right?

**Reply by 6r1d** [reply]
> Если они готовы заткнуть рот своим сторонникам, закрыв голосовалку - это их право, right?

Верно

> А без доказательств, все вытекающие из этого заявления, последствия подпадают под УК.

Ну, пока правовые вопросы поднимал лишь Крис. Я не до конца понимаю тему, чтобы о чём-то здесь говорить.

**Reply by AnimusPEXUS** [reply]
они в закрытую это делают теперь
ну и ещё, чтобы имена не палить, чтобы мы не сделали свои списки прокажённых

**Comment by 6r1d** [comment]
> Или это у них так 1 апреля отметили)

А кто их знает? Всё это напоминает непрерывное первое апреля.

**Comment by ZhymabekRoman** [comment]
Доброго времени суток. Мда уж, дела не свосем хорошие. Те кто не поддерживают Столмана решили сделать расширение для браузера которая помечает страницы людей, которые подписались в поддержку Столмана. Одним словом они решили устроить на нас охоту. Мало этого они еще и делают всем подписавшим за Столмана анонимную рассылку, чтобы удалили свою подпись из письма. Пруфы: https://www.mail-archive.com/gcc@gcc.gnu.org/msg94547.html

**Reply by AnimusPEXUS** [reply]
пусть **ахнут себя в зад. мне плевать на их расширения, рассылки и прочее.

**Reply by systemdfan** [reply]
> пусть **ахнут себя в зад. мне плевать на их расширения, рассылки и прочее.

Вот-вот. Ничего они нам не сделают. Пусть хоть наотмечаются.

**Comment by ZhymabekRoman** [comment]
Надо что-то делать, есть тут кто готов сделать расширение для браузера которая будет помечать всех людей которые подписалсь против Столмана? Давайте объединимся  и сделаем такое расширение, я готов занятся тестированием.

**Reply by 6r1d** [reply]
> есть тут кто готов сделать расширение для браузера которая будет помечать всех людей которые подписалсь против Столмана

Уже пробовали, и я не поддерживаю этот подход.

* Первое - многие противники РМС, несмотря на высокий статус, не всегда являются технарями и теми, с кем мы столкнёмся. Это расширение __будет совершенно бесполезным__, но __поддержит их подход__.
* Второе - их расширение является полнейшей чушью, которую впаривают другим.
* Третье - силы здесь надо потратить не на программирование, а на то, чтобы донести, какая это чушь. И это необязательно делать с криками и жалобами, надо обсуждать, как быть тактичными, выкладывать видео на Ютуб, высмеивать это мемами.

**Reply by dimqua** [reply]
Дело тут совсем не в этом, а в том, что это попросту не этично. Зачем их вообще отмечать, чтобы что? Травить и называть идиотами?

**Reply by AnimusPEXUS** [reply]
меня больше беспокоит как обстоят дела с ретрошарой, Дим. потому что, я чувствую, нас могут прикрыть в гитхабе - и тогда будет совсем негде каллаборироваться.. дискорд считать за подходящее место для чаттинга - ну это совсем блин. ретрошара лучше, даже при наличии сарки и гифок с жопами и фаллосами, благо их можно скрыть.

**Reply by AnimusPEXUS** [reply]
а что если анти-Сталлман движение - это заговор правительства? пу, ведь, поддержал Байдена. так что и нам может аукнуться тоже

**Reply by ghost** [reply]
Я вас умоляю. Аудитория того дополнения - 40 пользователей на Greasyfork. Единственный эффект, который оно способно произвести - это чувство глубокого самоудовлетворения у его автора. Кто сам-то захочет дружить (или не дай бог работать) с людьми, которые стреляют прежде чем задавать вопросы? Какие-то более серьёзные последствия для подписавшихся мне представляются маловероятными. Прежде всего потому, что это частная инициатива без какого-то серьёзного пиар-потенциала. 
Закрывать этот гитхаб тоже вряд ли будут. Таким образом Микрософт фактически распишется в своей заинтересованности, и если они не сделали этого раньше, то вряд ли сделают это сейчас. Почитайте письмо Лии Роу, она уже очень долго варится в теме свободного ПО и толково всё расписала. На крайний случай есть зеркала, есть вебархив. Если речь идёт о создании ещё одного зеркала - то оно, само собой, будет не лишним.
Самое важное, что на нашей стороне хороший этический имидж. Он, вместе с численностью, является нашим капиталом и важно не растерять его попусту. Потому что если ещё и в Дебиане примут одно из решений не против Столлмана, то это и вовсе будет повод для других проектов задуматься - а не поспешили ли они со своим осуждением.
А что до заговоров, то тут всё ещё проще. СЖВ - это не какие-то пришельцы с планеты Жопа. Это просто люди, и ими движет прежде всего страх. В частности, страх получить ненависть, неприятие. Нечистые на руку люди используют этот страх для своей выгоды. Кто это, я не могу сказать. Но будь у них действительно большое влияние, они бы использовали более мощные и верные методы. И я не рекомендую вам самим поддаваться страху и выдумывать себе бабаек, которые утащат вас за то, что вы где-то там подписались. Это никому не поможет и только сделает ситуацию хуже.
Кстати, что за Дискорд? Активно писать не обещаю, но хотелось бы быть в курсе событий.

**Reply by AnimusPEXUS** [reply]
@checooh https://github.com/rms-support-letter/rms-support-letter.github.io/issues/1863

**Comment by pda0** [comment]
Кто-нибудь ещё такое письмо получал?

A Condorcet Internet Voting Service poll named Holocaust Memorial today, Debian defending Nazis, Say No has been created. You have been designated as a voter by the poll supervisor, Yom HaShoah (Holocaust Memorial) (adolfhitler@netc.eu).

Description of poll: Today is Yom HaShoah, the day that Israel reflects on the Holocaust. Debian Developers are arguing that it is OK to be a Nazi. Meanwhile, Debian is actively voting on whether to publicly defame Dr Richard Stallman, who is of Jewish descent. Is it time for Debian to stop this obscene behavior, cancel the vote and delete all the history of defamation against volunteers from all Debian servers?

**Reply by AnimusPEXUS** [reply]
https://github.com/rms-support-letter/rms-support-letter.github.io/issues/6450
я тоже получал, если что

**Reply by Bluesboy** [reply]
Многие получили, можешь игнорировать.

**Reply by pda0** [reply]
> Многие получили, можешь игнорировать.

Я понимаю, просто интересен охват был.

**Reply by 6r1d** [reply]
Я тоже. Спам от Адольфа. Не ожидал, не ожидал.

**Reply by systemdfan** [reply]
А я знал что эта усатая тварь жива и прячется в лесах Аргентины! Письмо, кстати, я не получал. А оно приходит только тем кто адрес почты вместо сайта указал или как?

**Reply by 6r1d** [reply]
> А оно приходит только тем кто адрес почты вместо сайта указал или как?

Нет, это дружелюбный Git добавляет твой e-mail к коммитам, чтобы тебя потом можно было спросить о том, что ты написал 20 лет назад. :D

**Reply by systemdfan** [reply]
> Нет, это дружелюбный Git добавляет твой e-mail к коммитам, чтобы тебя потом можно было спросить о том, что ты написал 20 лет назад. :D

Пхах, а мой email тоже прикреплён получается? Странно, мне ничего не пришло опять-таки.

**Comment by 6r1d** [comment]
> Пхах, а мой email тоже прикреплён получается? Странно, мне ничего не пришло опять-таки.

Проверь папку спама

**Reply by systemdfan** [reply]
> > Пхах, а мой email тоже прикреплён получается? Странно, мне ничего не пришло опять-таки.
> 
> Проверь папку спама

Проверил - пусто.

**Reply by 6r1d** [reply]
> Проверил - пусто.

О, круто, адрес забанили до того, как спамили тебя. Есть польза от этих дискуссий.

**Comment by 6r1d** [comment]
У кого-нибудь есть контакты [Александры Элбакян](https://vk.com/sci_hub)? Я думаю, её голос за Ричарда тоже будет весомым, поскольку Sci-Hub — штука громкая и я ожидаю, что она поддержит нас.

**Reply by systemdfan** [reply]
> У кого-нибудь есть контакты [Александры Элбакян](https://vk.com/sci_hub)? Я думаю, её голос за Ричарда тоже будет весомым, поскольку Sci-Hub — штука громкая и я ожидаю, что она поддержит нас.

Ну вот беглым взглядом сразу нашёл skype: alexandra.elbakyan, почту: alexandra@dns.cymru, [PGP ключ ](https://sci-hub.do/alexandra#pgp), TOX ID: `9E9A53177552B8272FAFF91562C04C712CDC087EBD23155A76AEAB3EA435781369290D4611F1`

**Reply by 6r1d** [reply]
> сразу нашёл skype

А кто-то им пользуется, чтобы написать? Или Токсом? Почту в открытом виде выкладывать не стоит, даже чужую, спамботы не дремлют.

**Reply by systemdfan** [reply]
> > сразу нашёл skype
> 
> А кто-то им пользуется, чтобы написать? Или Токсом?

Ну я скайпом не пользуюсь. Почту, думаю она читает. Я лично свою каждый день проверяю.

> Почту в открытом виде выкладывать не стоит, даже чужую, спамботы не дремлют.

А как ей ещё поделиться в не открытом виде?

**Reply by dimqua** [reply]
@systemdfan написал. Ждем ответа.

**Reply by 6r1d** [reply]
> А как ей ещё поделиться в не открытом виде?

Я обитаю в Matrix, Discord и частенько на IRC. Боты читают все страницы и вытаскивают почтовые адреса. Большая часть спама останется спамом.

> написал. Ждем ответа.

Ура, это многое изменит, я так думаю!
